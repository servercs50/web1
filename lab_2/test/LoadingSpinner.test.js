import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import LoadingSpinner from "../src/components/LoadingSpinner";


describe('Loading spinner', () => {

  it('renders correctly', () => {
    const tree = shallow(<LoadingSpinner/>);
    expect(shallowToJson(tree)).toMatchSnapshot();
  });

});
