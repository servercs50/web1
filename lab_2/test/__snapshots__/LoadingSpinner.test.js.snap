// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading spinner renders correctly 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
  justify="center"
>
  <WithStyles(ForwardRef(Grid))
    container={true}
    justify="center"
  >
    <WithStyles(ForwardRef(Typography))
      align="center"
      component="h3"
      gutterBottom={true}
      variant="h5"
    >
      Подождите, данные загружаются
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    container={true}
    justify="center"
  >
    <Loader
      color="#123abc"
      css={Object {}}
      loading={true}
      size={60}
      sizeUnit="px"
    />
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
